app/views/users/index.html.erb

```diff
@@ -1 +1,17 @@
-<h1> Welcome to Users List</h1>
\ No newline at end of file
+<div class = 'user'>
+<h1> Welcome to Users List</h1>
+<div class = 'user_container'>
+<% @users.each do |user|%>
+<div class = 'user_row'>
+<div class = 'user_image'>
+
+</div>
+<div class = 'user_block'>
+<h3><%= link_to user.name, user_path(user)  %></h3>
+<img src=<%= user.photo %> >
+<p>Number of Posts = <%=user.posts.count %></p>
+</div>
+</div>
+<% end %>
+</div>
+</div>
```

Comment:
- [ ] [OPTIONAL] Well done for showing the image as requested by the previous reviewer 👏. Please take a look at the wireframe for the users index page. The layout for the image is on the left while the rest of the information is on the right. Please modify your design to follow the wireframe layout 🙏 
### Wireframe
![image](https://github.com/imranhossainemi/blog-website/assets/46839548/37319a59-5f9d-440c-88aa-1609828a6e37)
### Your Design
![image](https://github.com/imranhossainemi/blog-website/assets/46839548/be1b376c-59a7-4818-abea-fc02c4fdf932)